go.sum

```diff
@@ -754,6 +754,7 @@ golang.org/x/lint v0.0.0-20190227174305-5b3e6a55c961/go.mod h1:wehouNa3lNwaWXcvx
 golang.org/x/lint v0.0.0-20190301231843-5614ed5bae6f/go.mod h1:UVdnD1Gm6xHRNCYTkRU2/jEulfH38KcIWyp/GAMgvoE=
 golang.org/x/lint v0.0.0-20190313153728-d0100b6bd8b3/go.mod h1:6SW0HCj/g11FgYtHlgUYUwCkIfeOF89ocIRzGO/8vkc=
 golang.org/x/lint v0.0.0-20190409202823-959b441ac422/go.mod h1:6SW0HCj/g11FgYtHlgUYUwCkIfeOF89ocIRzGO/8vkc=
+golang.org/x/lint v0.0.0-20190909230951-414d861bb4ac h1:8R1esu+8QioDxo4E4mX6bFztO+dMTM49DNAaWfO5OeY=
```

Comment:
not sure, just did make test as usual.

Reply:
Won't be a problem since I'll rebase it as part of https://github.com/kiegroup/kogito-cloud-operator/pull/157

Reply:
@ricardozanini should it be included into the PR ?

Reply:
I'll merge this one first and rebase with mine. The SUM will change eventually since a lot of our dependencies have changed.

I won't merge mine first to not put more work on @spolti shoulders. Actually, I intend to merge all PRs before mine, though.